packages/dom/test/serialize-frames.test.js

```diff
@@ -60,6 +62,27 @@ describe('serializeFrames', () => {
     }
   }, 0); // frames may take a bit to load
 
+  afterEach(() => {
+    window.document.createDocumentFragment.calls.reset();
+  });
+
+  it('calls document.createDocumentFragment once for parent frame', () => {
+    // document.createDocumentFragment is only called on first depth of recursion for serializing iframes
+    // post that the document of the iframe itself should be used for creating document fragment
+    // we're having expection for root frame only currently, this should suffice for now
+
+    let timesCalled = 0;
+    if (getTestBrowser() === chromeBrowser) {
+      // we use plain & shadow platform
+      timesCalled = 2;
+    } else if (getTestBrowser() === firefoxBrowser) {
+      // we use only plain platform
+      timesCalled = 1;
+    }
+
+    expect(window.document.createDocumentFragment).toHaveBeenCalledTimes(timesCalled);
+  });
```

Comment:
Can we have the test check for the base uri's instead? This one doesn't seem very intuitive